solutions/sapo-14-genghis.mdx

```diff
@@ -0,0 +1,241 @@
+---
+id: sapo-14-genghis
+source: SAPO 2014
+title: The Stables of Genghis Khan
+author: Kevin Sheng
+---
+
+Let's define $\texttt{lowestOps}[a][b]$ as the minimum number of operations to see if each horse is in the stables or not, given that we only consider horses in the interval $(\texttt{stableHorses[a]}, \texttt{stableHorses[b]})$, with $\texttt{stableHorses}$ being a sorted list of all the horses that are in the stables.
+
+Why use an open interval? It's to account for the horses with ID's that aren't contained in the range of the ID's of the horses of the stable. For example, if we had a horse with ID $4$, and the horses in the stables were $[6, 7, 10]$, then there would be no way for $\texttt{lowestOps}$ to account for that horse.
```

Comment:
maybe put this in an info block

Reply:
the why use an open interval part

Reply:
think it's important enough to be in the editorial